src/registrar/management/commands/copy_names_from_contacts_to_users.py

```diff
@@ -0,0 +1,248 @@
+
+    # C901 'Command.handle' is too complex
+    def process_contacts(
+        self,
+        debug_on,
+        skipped_contacts,
+        eligible_users,
+        processed_users,
+    ):
+        for contact in Contact.objects.all():
+            # DEBUG:
+            TerminalHelper.print_conditional(
+                debug_on,
+                f"{TerminalColors.OKCYAN}"
+                "Processing Contact: "
+                f"{contact.email},"
+                f" {contact.first_name},"
+                f" {contact.last_name}"
+                f"{TerminalColors.ENDC}",  # noqa
+            )
+
+            # ======================================================
+            # ====================== USER  =======================
+            (eligible_user, processed_user) = self.update_user(contact, debug_on)
+
+            debug_string = ""
+            if eligible_user:
+                # ---------------- UPDATED ----------------
+                eligible_users.append(contact.email)
+                debug_string = f"eligible user: {eligible_user}"
+                if processed_user:
+                    processed_users.append(contact.email)
+                    debug_string = f"processed user: {processed_user}"
+            else:
+                skipped_contacts.append(contact.email)
+                debug_string = f"skipped user: {contact.email}"
+
+            # DEBUG:
+            TerminalHelper.print_conditional(
+                debug_on,
+                (f"{TerminalColors.OKCYAN} {debug_string} {TerminalColors.ENDC}"),
+            )
+
+        return (
+            skipped_contacts,
+            eligible_users,
+            processed_users,
+        )
+
+    # ======================================================
+    # ===================== HANDLE  ========================
+    # ======================================================
+    def handle(
+        self,
+        **options,
+    ):
+        """Parse entries in Contact table
+        and update valid corresponding entries in the
+        User table."""
+
+        # grab command line arguments and store locally...
+        debug_on = options.get("debug")
+
+        self.print_debug_mode_statements(debug_on)
+
+        # users we SKIPPED
+        skipped_contacts = []
+
+        # users we found that are linked to contacts
+        eligible_users = []
+
+        # users we PROCESSED
+        processed_users = []
+
+        logger.info(
+            f"""{TerminalColors.OKCYAN}
+            ==========================
+            Beginning Data Transfer
+            ==========================
+            {TerminalColors.ENDC}"""
+        )
+
+        logger.info(
+            f"""{TerminalColors.OKCYAN}
+            ========= Adding Domains and Domain Invitations =========
+            {TerminalColors.ENDC}"""
+        )
+        (
+            skipped_contacts,
+            eligible_users,
+            processed_users,
+        ) = self.process_contacts(
```

Comment:
**comment (non-blocking)**: I think that this signature is confusing. It takes the (empty) lists as arguments so that it can append to them, but then it returns the appended lists as well? Why not one or the other? 

Either take no lists as arguments and return the lists from inside the function, or take the list objects to be added to as arguments but then you don't need to return them. I personally think the first option of no arguments and just returning the lists is most pythonic. Then you won't have to initialize the empty lists in this calling function.